src/components/fileHandler.ts

```diff
@@ -0,0 +1,76 @@
+import path from 'path';
+import fs from 'fs';
+import util from 'util';
+import { pipeline } from 'stream';
+import getAppRootDir from './getAppRoot';
+import { pgClient } from './postgresConnect';
+import * as config from '../config.json';
+
+export const filesFolderName = config.filesFolderName;
+
+export function createFilesFolder() {
+  try {
+    fs.mkdirSync(path.join(getAppRootDir(), filesFolderName));
+  } catch {
+    // Folder already exists
+  }
+}
+
+export async function getFilename(id: string) {
+  const result = await pgClient.query('SELECT path, original_filename FROM file WHERE id = $1', [
+    id,
+  ]);
+  const folder = result.rows[0].path; // Not currently used
+  const filenameOrig = result.rows[0].original_filename;
+  const ext = path.extname(String(filenameOrig));
+  const basename = path.basename(filenameOrig, ext);
+  return `${basename}_${id}${ext}`;
+}
+
+const pump = util.promisify(pipeline);
+export async function saveFiles(data: any, queryParams: any) {
```

Comment:
Types for `data` and `queryParams` please

Reply:
Have defined an interface for `queryParams`, but how would I go about defining the type for `data`? -- it's a collection of files passed from an http POST request. 🤷‍♂️

Reply:
Yea. Fair enough. We can check later if there is any way to have a type for `data`.